src/LexerParser/Parser.cpp

```diff
@@ -69,6 +72,35 @@ namespace JDD::Parser {
         std::cout << content;
     }
 
+     void JDDParser::assert_eq(std::vector<Lexer::Token>::const_iterator &current, Definition::Data& data) {
+        auto openValues = ExpectOperator(current, "(");
+        if (!openValues.has_value())
+            std::cerr << "Forgot to open with '(' and make sure to close with ')'" << std::endl;
+
+        auto value = ExpectValue(current, data);
+        if (!value.has_value())
+            std::cerr << "Forgot to give value in print" << std::endl;
```

Comment:
Beware of copy and paste. Here, there is a reference to print instead of assert_eq